modules/keycloak/ssm.tf

```diff
@@ -81,3 +81,29 @@ resource "aws_ssm_parameter" "keycloak_user_admin_client_secret" {
     ignore_changes = [value]
   }
 }
+
+//No programmatical method to retrieve value from GoUk Notify service
+//Use Terraform to add the parameter and update value manually
+//As added by Terraform won't be removed when Terraform apply run
+resource "aws_ssm_parameter" "keycloak_govuk_notify_api_key" {
```

Comment:
Just a note to myself to come back to this once we've discussed https://github.com/nationalarchives/tdr-terraform-backend/pull/105/files#r603079753 - I'm still trying to wrap my head around how this parameter gets set in the different environments.